src/piadm.sh

```diff
@@ -355,6 +357,7 @@ EOF
 
 	# Can furthermore be overridden by the user's PIADM_URL_PREFIX.
 	URL_PREFIX=${PIADM_URL_PREFIX:-${DEFAULT_URL_PREFIX}}
+	source "${PIADM_CONF}"
```

Comment:
If I set one of the above in the environment, reading the file here will override, and that's not good. Maybe the file has entries like `PIADM_NO_SUM=${PIADM_NO_SUM:-0}` or something else to make sure user environment overrides the defaults in the file?